coderd/database/dbmem/dbmem.go

```diff
@@ -7275,13 +7276,7 @@ func (q *FakeQuerier) ListProvisionerKeysByOrganization(_ context.Context, organ
 	keys := make([]database.ProvisionerKey, 0)
 	for _, key := range q.provisionerKeys {
 		if key.OrganizationID == organizationID {
-			keys = append(keys, database.ProvisionerKey{
-				ID:             key.ID,
-				CreatedAt:      key.CreatedAt,
-				OrganizationID: key.OrganizationID,
-				Name:           key.Name,
-				HashedSecret:   key.HashedSecret,
-			})
+			keys = append(keys, key)
```

Comment:
Maybe this was done in the past to avoid a copy by reference? 

Works fine as it is though :+1: 
https://goplay.tools/snippet/6f0EHnip_Ss

Reply:
It was a specific DB query row type before and now the query returns the whole type.